### PR TITLE
Control updates - Jan 24

### PR DESCRIPTION
**Pull request by Aboli-msft**

### CODE REVIEW

File: Control coverage/README.md [hunk 1]
```diff
@@ -45,6 +46,7 @@ Below resource types can be checked for validating the security controls:
 |[SQLServer](Feature/SQLServer.md)|Microsoft.Sql/servers|
 |[Storage](Feature/Storage.md)|Microsoft.Storage/storageAccounts|
 |[Subscription](Feature/SubscriptionCore.md)|
+|[SynapseWorkspace](Feature/SynapseWorkspace.md)|
```
[the remark this answers is not in the document]
Comment (Aboli-msft) [reply Control coverage/README.md]: Added

File: Control coverage/Feature/SynapseWorkspace.md [hunk 2]
```diff
@@ -0,0 +1,199 @@
+# SynapseWorkspace
+
+**Resource Type:** Microsoft.Synapse/workspaces
+
+<!-- TOC -->
+
+- [Azure_SynapseWorkspace_AuthN_SQL_Pools_Use_Microsoft_Entra_ID_Only](#azure_synapseworkspace_authn_sql_pools_use_microsoft_entra_id_only)
+- [Azure_SynapseWorkspace_Audit_Enable_Diagnostics_Log](#azure_synapseworkspace_audit_enable_diagnostics_log)
+- [Azure_SynapseWorkspace_AuthN_SQL_Pools_Use_Microsoft_Entra_ID_Only](#azure_synapse_netsec_dont_allow_universal_ip_range)
```
[the remark this answers is not in the document]
Comment (Aboli-msft) [reply Control coverage/Feature/SynapseWorkspace.md]: Corrected

File: Control coverage/Feature/SynapseWorkspace.md [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+
+### Azure Policies or REST APIs used for evaluation 
+
+- REST API used to list Synapse workspaces at Subscription level: <br />
+/subscriptions/{subscriptionId}/providers/Microsoft.Synapse/workspaces?api-version=2021-06-01<br />
+
+- REST API used to list Firewall rules for Synapse workspace:
+/subscriptions/{subscriptionId}/resourceGroups/{1}/providers/Microsoft.Synapse/workspaces/{synapseWorkspaceName}/firewallRules?api-version=2021-06-01
+
+**Properties:**
```
Comment (pranchalsomani) [inline Control coverage/Feature/SynapseWorkspace.md:171]: Please make these properties aligned with the above section.

**Review by pranchalsomani — CHANGES_REQUESTED** [review]

File: Control coverage/Feature/SynapseWorkspace.md [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
```
[the remark this answers is not in the document]
Comment (Aboli-msft) [reply Control coverage/Feature/SynapseWorkspace.md]: updated

File: Control coverage/Feature/MachineLearningWorkspace.md [hunk 5]
```diff
@@ -0,0 +1,300 @@
...
+> **Failed:**
+> 1. Diagnostics setting is disabled for resource.
+> 
+>       or
+>
+> 2. Diagnostic settings meet the following conditions:
+> a. All diagnostic logs are not enabled.
+> b. None of the below settings is configured:
+
+> i. Log Analytics.
```
[the remark this answers is not in the document]
Comment (Aboli-msft) [reply Control coverage/Feature/MachineLearningWorkspace.md]: updated

File: Control coverage/Feature/SynapseWorkspace.md (hunk 3 above)
Comment (Aboli-msft) [reply Control coverage/Feature/SynapseWorkspace.md]: updated

File: Control coverage/Feature/MachineLearningWorkspace.md [hunk 6]
```diff
@@ -0,0 +1,300 @@
...
+```json 
+{
+  "CoveredComputeTypes": [
+    "ComputeInstance",
+    "AmlCompute"
+  ]
+}
+```
+
+### Control Spec
```
[the remark this answers is not in the document]
Comment (Aboli-msft) [reply Control coverage/Feature/MachineLearningWorkspace.md:131]: Updated

File: Control coverage/Feature/SynapseWorkspace.md [hunk 7]
```diff
@@ -0,0 +1,176 @@
...
+>Custom firewall rule with Any-to-Any IP range with Start IP address as 0.0.0.0 and End Ip address as 255.255.255.255 is NOT found.
+>Firewall rule that gets added from "Allow Azure resources to access this resource" is allowed.
+
+> **Failed:**
+> Custom firewall rule with Any-to-Any IP range with Start IP address as 0.0.0.0 and End Ip address as 255.255.255.255 is found.
+
+
+### Recommendation 
+
+- **Azure Portal**
```
[the remark this answers is not in the document]
Comment (Aboli-msft) [reply Control coverage/Feature/SynapseWorkspace.md:155]: updated

File: Control coverage/Feature/VirtualNetwork.md [hunk 8]
```diff
@@ -656,8 +656,10 @@ Restricting inbound and outbound traffic via NSGs limits the network exposure of
     ],
     "SubnetsToExcludeFromEvaluation": [
         "azurefirewallsubnet",
-	"gatewaysubnet",
-        "routeserversubnet"
+		"azurefirewallmanagementsubnet",
+		"gatewaysubnet",
+        "routeserversubnet",
```
[the remark this answers is not in the document]
Comment (Aboli-msft) [reply Control coverage/Feature/VirtualNetwork.md]: fixed

**Review by pranchalsomani — APPROVED** [review]

**Review by jaiswalricha — APPROVED** [review]

**Pull request merged**